model/UserStatus.ts

```diff
@@ -0,0 +1,9 @@
+export const userStatus = ["ACTIVE", "INACTIVE", "DELETED", "PASSWORD_EXPIRED"] as const
```

Comment:
why need `as const`?
can we write like `export const userStatus: string[] = ["ACTIVE", "INACTIVE", "DELETED", "PASSWORD_EXPIRED"]`